__init__.py

```
@@ -1,13 +1,17 @@
+from pathlib import Path
 from ast import literal_eval as parse_tuple
 from ovos_color_parser import color_from_description, sRGBAColor
+from ovos_color_parser.matching import lookup_name
 from ovos_utils import create_daemon, classproperty
 from ovos_utils.log import LOG
 from ovos_utils.process_utils import RuntimeRequirements
-from ovos_workshop.decorators import intent_handler
+from ovos_workshop.decorators import intent_handler, resting_screen_handler
 from ovos_workshop.intents import IntentBuilder
 from ovos_workshop.skills import OVOSSkill
+from ovos_workshop.skills.api import SkillApi
```

Ruff (0.8.2)

`ovos_workshop.skills.api.SkillApi` imported but unused

(F401)